src/components/Graph/Graph.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Node from './Node';
+import Edge from './Edge';
+
+const ListItem = ({ items, renderItem }) => {
+  return items.map((item, index) => renderItem(item, index));
+};
+
```

Comment:
The name for this indicate that this is the list item but this is the entire list. 
This should be the generic ListComponent (change the name)